doc/README.md

```diff
@@ -0,0 +1,103 @@
+There are two ways to achieve this:
+
+* Use the [kubectl-linstor](https://github.com/piraeusdatastore/kubectl-linstor) plugin. The plugin interfaces
+  with the resources used by the Piraeus Operator and enables you to use `kubectl linstor ...` to execute LINSTOR
+  commands.
+* Execute the `linstor` command directly in the controller pod:
+  ```
+  kubectl exec -it deployment/piraeus-op-cs-controller -- linstor ...
+  ```
+
+## Configuring storage pools
+
+To provision volumes, you need to configure storage pools. The LINSTOR backend supports
+a range of different storage providers. For some common providers, the Piraeus Operator
+provides convenient configuration via the `LinstorSatelliteSet` resource. You can read more
+on how to configure storage [here](./storage.md)
+
+## Creating volumes
+
+Once you have storage pools configured (confirm by running `kubectl linstor storage-pool list`), you
+can almost start creating Persistent Volumes (PV) with Piraeus. First you will need to create
+a new storage class in Kubernetes.
+
+The following example storage class configures piraeus to:
+* use 2 replicas for every persistent volume
+* use the `xfs` filesystem
+* use storage pools named `ssd`
+* allow volume expansion by resizing the Persistent Volume Claim (PVC).
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: piraeus-ssd
+provisioner: linstor.csi.linbit.com
+allowVolumeExpansion: true
+parameters:
+  autoPlace: "2"
+  storagePool: ssd
+  csi.storage.k8s.io/fstype: xfs
+```
+
+You can find a full list of supported option [here](https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-kubernetes-sc-parameters).
+
+Using this storage class, you can provision volumes by applying a Persistent Volume Claim and waiting
+for Piraeus to provision the PV. The following PVC creates a 5GiB volume using the above storage class:
+
+```yaml
+apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: piraeus-pvc-1
+  spec:
+    storageClassName: piraeus-ssd
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 5Gi
+```
+
+## Snapshots
+
+Piraeus supports snapshots via the CSI snapshotting feature. To enable this feature in your
+cluster, you need to add a [Snapshot Controller](https://github.com/kubernetes-csi/external-snapshotter/) to your cluster.
+
+Some Kubernetes distributions (for example: OpenShift) already bundle this snapshot controller. On distributions
+without a bundled snapshot controller, you can use our guide [here](./optional-components.md#snapshot-support-components).
+
+# Additional information
+
+Should you require further information check out the following links:
+
+* Additional documentation on [LINSTOR](https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-kubernetes-basic-configuration-and-deployment)
+
+If you run into issue, don't hesitate to open an issue on [GitHub](https://github.com/piraeusdatastore/piraeus-operator/issues)
```

Comment:
```suggestion
If you run into issues, don't hesitate to open an issue on [GitHub](https://github.com/piraeusdatastore/piraeus-operator/issues)
```

Do we want to invite issues quite so strongly? Issues should really be for when it looks like there is a problem with the operator. For getting help, users should use Slack or the mailing list, or become a LINBIT customer. I think it is perfectly reasonable to mention the option of paid support here.

Reply:
:+1: